Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -84,9 +89,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $em->getConnection()->connect($input->getOption('shard'));
         }
 
-        $fixtures = $this->fixturesLoader->getFixtures();
+        $dirOrFile = $input->getOption('fixtures');
+        if ($dirOrFile) {
+            $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+            $loader = new DataFixturesLoader($this->getContainer());
```

Comment:
As previously indicated, falling back to directories is a bad idea IMO and should be changed. I've suggested using an attribute on the fixtures tag to configure the fixtures set and using that to determine fixture sets - I believe that is still the way to go. Do you want to give that a shot?

Reply:
it would be great if one fixture class could be in multiple groups. Something like `sets={"set1","set2","set3"}` or maybe passing the service ids as argument and not the path?